rt/rs/security/oauth-parent/oauth2/src/main/java/org/apache/cxf/rs/security/oauth2/utils/OAuthUtils.java

```diff
@@ -429,6 +429,6 @@ public static SignatureAlgorithm getClientSecretSignatureAlgorithm(Properties si
     }
 
     public static String convertListOfScopesToString(List<String> registeredScopes) {
-        return String.join(", ", registeredScopes);
+        return String.join(" ", registeredScopes);
```

Comment:
What's the reason behind this change?

Reply:
https://tools.ietf.org/html/rfc7591#section-4.1.2

>    o  Client Metadata Name: "scope"
>    o  Client Metadata Description: Space-separated list of OAuth 2.0
>       scope values